docs/reference/modules/threadpool.asciidoc

```diff
@@ -75,7 +75,7 @@ requests with a queue (optionally bounded) for pending requests that
 have no threads to service them.
 
 The `size` parameter controls the number of threads, and defaults to the
-number of cores times 5.
+number of cores.
```

Comment:
Let's revert this change, I do not think it's correct anyway any longer, there is no default for a fixed thread pool, it must be specified on construction of the thread pool and each thread pool is free to set its own default. Fixing this should be done in a separate change.